### PR TITLE
Local changes

### PR DESCRIPTION
**Pull request by joaobrittoneto**
Fix dynamic properties

### CODE REVIEW

File: tasks/Task.cpp [hunk 1]
```diff
@@ -97,7 +98,9 @@ bool Task::configureHook()
     driver.reset(new Driver());
     if (!_io_port.get().empty())
         driver->openURI(_io_port.get());
+    std::cout << "openURI" << std::endl;
```
Comment (doudou) [inline tasks/Task.cpp]: leftover debug output ...

**Comment by joaobrittoneto** [comment]
@doudou, don't merge now.
I'll make some changes, treat exception that are happens (FULL_QUEUE) and may happens (HUGE_DATA)

**Comment by joaobrittoneto** [comment]
@doudou, 
Do you have any comment here? May I merge?

**Comment by doudou** [comment]
Nope. Looks good.

**Pull request merged**